CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.1.0] - [2017-09-22]
```

Comment:
remove version, this is done by maintainers, see: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md for more information and examples.

Reply:
I have removed the version.